tasks/sep/fp-recovery/004-recover-bonds/input-template.json

```diff
@@ -0,0 +1,33 @@
+{
+  "chainId": 11155111,
+  "metadata": {
+    "name": "Guardian - Recover all funds in the DelayedWETH contract",
+    "description": "This task calls the `recover` function in DelayedWETH to retrieve the funds locked in the DelayedWETH contract. This task should be executed when the Superchain is paused to avoid ongoing games from funding the contract"
+  },
+  "transactions": [
+    {
+      "metadata": {
+        "name": "DelayedWETH.recover",
+        "description": "Recover all funds in DelayedWETH"
+      },
+      "to": "0xca36b00f885d83068f0cffee381f0c974de1d58e",
```

Comment:
Looks to be an out of date `DelayedWETH` deployment: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/extra/addresses/sepolia/op.json#L14